File: Solutions/458_Poor_Pigs/458_Poor_Pigs.c
int poorPigs(int buckets, int minutesToDie, int minutesToTest) {return (int)ceil(log(buckets) / (log(minutesToTest / minutesToDie + 1)));}

//Another
int poorPigs(int buckets, int minutesToDie, int minutesToTest){
    int i =  minutesToTest/minutesToDie + 1;
    int n = 0;
    int t = 1;
    while (t < buckets) {
        t = i * t;
        n++;
    }
    return n;
}

File: Solutions/458_Poor_Pigs/458_Poor_Pigs.go
func poorPigs(buckets int, minutesToDie int, minutesToTest int) int {
    testRound := minutesToTest / minutesToDie
    testGroup := testRound + 1
    res := int(math.Ceil(math.Log(float64(buckets)) / math.Log(float64(testGroup))))
    return res
}